test/integration/admin_can_view_all_orders_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class AdminCanViewAllOrdersTest < ActionDispatch::IntegrationTest
+  test 'as an admin I can view all orders from users.' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.